src/main/kotlin/org/jetbrains/research/ml/ast/cli/Runner.kt

```diff
@@ -0,0 +1,123 @@
+package org.jetbrains.research.ml.ast.cli
+
+import com.intellij.ide.impl.ProjectUtil
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.application.ApplicationStarter
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiFileFactory
+import com.jetbrains.python.PythonFileType
+import com.xenomachina.argparser.ArgParser
+import org.jetbrains.research.ml.ast.transformations.Transformation
+import org.jetbrains.research.ml.ast.util.createFile
+import org.jetbrains.research.ml.ast.util.createFolder
+import org.jetbrains.research.ml.ast.util.getContentFromFile
+import org.jetbrains.research.ml.ast.util.getFilesFormFolder
+import java.io.File
+import java.nio.file.Paths
+import kotlin.random.Random.Default.nextDouble
+import kotlin.system.exitProcess
+
+// TODO: test it with different transformations sets
+object Runner : ApplicationStarter {
+    private lateinit var inputDir: String
+    private lateinit var outputDir: String
+    private lateinit var yaml_config: String
+
+    private var project: Project? = null
+    private val logger = Logger.getInstance(this::class.java)
+
+    override fun getCommandName(): String = "python-transformations"
+
+    class TransformationsRunnerArgs(parser: ArgParser) {
+        val input by parser.storing(
+            "-i",
+            "--input_path",
+            help = "Input directory with python files"
+        )
+
+        val output by parser.storing(
+            "-o",
+            "--output_path",
+            help = "Output directory"
+        )
+
+        val yaml by parser.storing(
+            "-y",
+            "--yaml_path",
+            help = "YAML config path"
+        )
+    }
+
+    private fun getTmpProjectDir(): String {
+        val path = "${System.getProperty("java.io.tmpdir")}/tmpProject"
```

Comment:
Maybe we can use some more meaningful name than "tmpProject"? How about "ast-transformations-tmp"?